apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/project.json

```diff
@@ -26,6 +26,6 @@
   },
 
   "frameworks": {
-    "netcoreapp1.6": {}
+    "netstandard1.6": {}
```

Comment:
Can we target netstandard1.5, or is there something we need from 1.6? (We're using 1.5 in most places)